src/hidimstat/utils.py

```diff
@@ -323,14 +323,19 @@ def _alpha_max(X, y, use_noise_estimate=False):
     """
     n_samples, _ = X.shape
 
-    if not use_noise_estimate:
-        alpha_max = np.max(np.dot(X.T, y)) / n_samples
-    else:
+    # compute the Gram matrix
+    gram = np.dot(X.T, y)
```

Comment:
Regarding the naming, this is only a gram matrix if y=X

Reply:
I changed the name. I am not very happy with the new name 'Xt_y'. I will be happy to have some suggestions.

Reply:
It seems that in sklearn they did not come up with a much better name. That looks good to me. 
https://github.com/scikit-learn/scikit-learn/blob/98ed9dc73a86f5f11781a0e21f24c8f47979ec67/sklearn/linear_model/_coordinate_descent.py#L95